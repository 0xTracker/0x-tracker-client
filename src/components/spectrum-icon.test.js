import { render } from 'react-testing-library';
import React from 'react';

import SpectrumIcon from './spectrum-icon';

describe('spectrum icon component', () => {
  it('should render without props', () => {
    const { container } = render(<SpectrumIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
