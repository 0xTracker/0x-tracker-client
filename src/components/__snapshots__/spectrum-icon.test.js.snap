// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spectrum icon component should render without props 1`] = `
.c0 {
  color: inherit;
}

.c0 path {
  fill: currentColor;
}

<svg
  class="c0"
  viewBox="0 0 15 15"
>
  <path
    d="M0 6.5V1c0-.6.4-1 1-1 9 .3 13.7 5 14 14 0 .6-.4 1-1 1H8.5c-.6 0-1-.4-1-1-.3-4.4-2-6.2-6.5-6.5-.6 0-1-.4-1-1z"
  />
</svg>
`;
